### PR TITLE
First stab at writing out the test_mission_data.py

### PR DESCRIPTION
**Pull request by pbvarga1**

### CODE REVIEW

**Comment by godber** [comment]
OK, this fails because the test tries to run and does not have `data.json`.  The question is now, should we run `get_mission_data` in CI?  We should probably make the test pass if `data.json` is not there.  Or maybe there is something better than "pass" like some dynamic pytest "skipped" option if `data.json` is not present.

**Comment by pbvarga1** [comment]
Or can we raise an error notifying the user to use ``get_mission_data`` so they are set up to test? My only worry of using ``get_mission_data`` by default is I don't want to download several gigabytes worth of data for the user. Ill look into the pytest "skipped"

**Comment by pbvarga1** [comment]
Should I a try using ``@pytest.mark.skipif`` first?

**Comment by godber** [comment]
Looks like exactly what I was thinking.

https://pytest.org/latest/skipping.html#marking-a-test-function-to-be-skipped

File: tests/test_mission_data.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import os
+from planetaryimage.pds3image import PDS3Image
+import json
+import pytest
+import pvl
+
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'mission_data/')
```
Comment (godber) [inline tests/test_mission_data.py]: Please remove the `/` at the end of `mission_data/`.

**Comment by godber** [comment]
This looks great.  I have some concerns about the label test however.  The use of `image.label.items()[0][1]` is not general enough.  I think we are going to need to change the `data.json` label entries a bit and modify this test.  How, is not quite clear to me yet.

**Pull request merged**